generate_report.py

```diff
@@ -0,0 +1,41 @@
+import csv
+import sys
+
+from src.routes_report import RoutesReport
+from src.search_count_report import SearchCountReport
+
+def gen_report(filename):
+  # Make sure filename is a csv file
+  extension_idx = filename.rfind('.csv')
+  if extension_idx == -1:
+    print(f'{filename} is not a csv file! Please try again')
+    return
+
+  # Read filename as csv
```

Comment:
should we move this to a separate method called `get_csv_reader`

Reply:
i don't think its necessary since we only do it once and its 2 lines